src/pages/catalog/index.ts

```diff
@@ -0,0 +1,73 @@
+import { createSearch } from '../../components/search';
+import { createFilter } from '../../components/filter';
+import Page from '../../types/page';
+import { createCatalogList } from '../../components/products-list';
+import { data } from '../../data/data';
+import { convertQuery } from '../../helpers/convert-query';
+import { routes } from '../../enums';
+import { FilterInterface } from 'components/filter/types';
+
+class CatalogPage extends Page {
+    constructor(id: string) {
+        super(id);
+    }
+
+    createCatalogPage(params: URLSearchParams) {
+        const paramPet = params.get('pet') || null;
+
+        console.log(params.entries());
+
+        const entries = params.entries();
+        let filterData = {} as FilterInterface;
+
+        for (const [key, value] of entries) {
+            filterData = { ...filterData, [key]: value.split(',') };
+        }
```

Comment:
Please, try to avoid `let`, I think you can find another way to get your `filterData`: maybe arr.reduce()

Reply:
`Entries` is an Object. I can't use reduce()

Reply:
Try to use Array.from()